### PR TITLE
2.5.3 preliminary changelog

### PR DESCRIPTION
**Pull request by Bob-Murphy**
Updates the changelog

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,91 @@
...
+  - Fixed vehicle pools not being properly saved #2106
+  - Vehicles near HQ now also have state preservation #2107
+  - Made HC squad vehicle placement use the garage placing code #2109
+  - Fixed broken object carrying #2109
+  - Fixed scaling and bugs plus added logging on economicsAI #2110
+  - Fixed Petros having no ammunition by giving him a vest #2111
+  - Fixed QRFs and singleAttacks being limited by incorrent maxUnits check #2113
+  - Fixed simulated attacks massively overfilling garrisons #2116
+  - Added more explanations to parameters #2120
+  - Fixed multiple issues regarding mortat type checking and locality #2125
```
Comment (jaj22) [inline changelog.txt:68]: typo, mortat->mortar

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,91 @@
...
+  - Fixed error in minefield saving #2127
+  - Fixed typos and punctuations in customHints #2131
+  - Fixed incorrect attack countdown incrementing #2135
+  - Fixed imbalance betweel the difficulty settings #2136
+  - Fixed bad exitWith in resourceCheck causing incorrect losses #2141
```
Comment (jaj22) [inline changelog.txt:73]: This was a fix for a new bug in 2.5.3 anyway, so can be stripped from the changelog.

Comment (Bob-Murphy) [reply changelog.txt:73]: the 2141?

Comment (jaj22) [reply changelog.txt:61]: Yeah, 2141.

File: changelog.txt [hunk 3]
```diff
@@ -1,3 +1,78 @@
...
+  - Moved singleAttack and patrolReinf logging to server #2102
+  - Fixed fastrope spawning corpses underground after being hit whilst fastroping #2103
+  - Fixed allowCrewInImmobile not being applied to convoy vehicles #2105
+  - Fixed vehicle pools not being properly saved #2106
+  - Vehicles near HQ now also have state preservation #2107
+  - Made HC squad vehicle placement use the garage placing code #2109
+  - Fixed broken object carrying #2109
+  - Fixed scaling and bugs plus added logging on economicsAI #2110
+  - Fixed Petros having no ammunition by giving him a vest #2111
+  - Fixed QRFs and singleAttacks being limited by incorrent maxUnits check #2113
```
Comment (jaj22) [inline changelog.txt:53]: typo, incorrent->incorrect

File: changelog.txt [hunk 4]
```diff
@@ -1,3 +1,78 @@
...
+  - Fixed degenerate behaviour in rebelAttack #2144
+  - Reduced capture response delay time #2147
+  - Fixed airborne troops being able to flip flags #2148
+  - Fixed exploit where commander could become permanently undercover #2149
+  - Fixed multiple bugs with squad/vehicle pricing #2151
+  - Added setOvercast functionality on rain-removal #2157
+  - Fixed lamp drop action not being added after respawn #2157
+  - Fixed not removing undercover status when placing ACE explosives #2158
+  - Fixed Nato gunship support #2160
+
```
Comment (killerswin2) [inline changelog.txt:71]:
```suggestion
  - Fixed garage feedback displaying on wrong clients  #2156
  
```

File: changelog.txt [hunk 5]
```diff
@@ -1,3 +1,78 @@
...
+================ Major ================
+  - Rework of punishment attacks #2119
+  --> Punishment attacks no longer sent against occupant-controlled towns.
+  --> Player scaling added to delay time and attacker vehicle count.
+  --> Improved vehicle choice and maxUnits control, ensure 2+ transports.
+  --> Defender ("civilian") count non-linearised, splits to groups of 4.
+  --> Removed civInit from defenders, so that they shoot and don't affect aggro.
+  --> Use combat landing in preference to fastrope, remove slow landing.
+  --> Simplified termination conditions.
+  --> Results only adjust support of nearby cities.
```
Comment (HakonRydland) [inline changelog.txt:14]: Results now only

File: changelog.txt [hunk 6]
```diff
@@ -1,3 +1,78 @@
...
+  --> Enable reinforcing from "carriers".
+  --> Separate road patrol generation from reinforcements and rebalance.
+  --> Sanitize garrison sizes (fewer giant and tiny garrisons, units of 4 rather than 8).
+  --> Use more 4-man teams in garrisons for both init and reinf.
+  --> Prevent new reinf convoys spamming after a recapture.
+  --> Fill out the AA & AT squads with a fourth soldier, cap militia squads to 8.
+  - Added player-count based balance for QRFs / singleAttack / wavedCA #2124
+
+================ Minor ================
+  - Garaging now only is possible at locations with flipable flags #2107
```
Comment (HakonRydland) [inline changelog.txt:29]: and hq, probably better to just say with flag poles

Comment (Bob-Murphy) [reply changelog.txt:29]: Roadblocks also have flagpoles, but they ain't working for garaging. Therefore the specific point. But yes, HQ is missing there.

**Review by jaj22 — APPROVED** [review]
I'd have slimmed down the descriptions of #2119 and #2121 because they were written for the benefit of reviewers rather than users, but it'll do.

**Pull request merged**